extensions/ui/org.eclipse.smarthome.ui.paper/web-src/partials/configuration.binding.html

```diff
@@ -0,0 +1,50 @@
+<section id="main" class="white-bg">
+	<div class="header-toolbar">
+		<md-button class="md-icon-button" aria-label="Refresh" ng-click="refresh()" aria-label="Refresh"> <md-tooltip>Refresh</md-tooltip> <i class="material-icons">refresh</i> </md-button>
```

Comment:
The `BindingDetailController` does not define a `refresh` method so this button actually does nothing. You might remove it (my suggestion) or implement the refresh method to reload the binding.